Vagrantfile

```diff
@@ -126,6 +126,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # Start an OpenShift cluster
     # Currently this only works with the (default) VirtualBox provider.
 
+    # Tag configuration as stale when provisioning a dev cluster to
+    # ensure that nodes can wait for fresh configuration to be generated.
+    if ARGV[0] =~ /^up|provision$/i and not ARGV.include?("--no-provision")
```

Comment:
vagrant up (without --no-provision) || vagrant provision

I was trying to avoid marking configuration as stale for any given vagrant invocation, and considered the vagrant cli to be very much stable for these fundamental operations.  If you're willing to live with marking configuration as stale for any given vagrant invocation (including formally read-only operations like 'status'), perhaps the conditional is unnecessary.